app/views/home/mln_calendar.html.erb

```diff
@@ -0,0 +1,10 @@
+<% unless @calendar_event.present? %>
+  <div role="main">
+   <h2 class="pg--title">Mylibrarynyc calendar</h2>
+    <div class="pg--content">
+      <table>
+        Mylibrarynyc calendar not found. Please create calendar.
```

Comment:
If this is the error that regular patrons see we should not say "Please create calendar." — you can just remove that second sentence.